src/tests/keys.rs

```diff
@@ -798,3 +799,225 @@ fn test_rsa_key_unwrap_vector() {
     );
     assert_eq!(ret, CKR_OK);
 }
+
+#[test]
+#[parallel]
+fn test_secret_key_attributes_matrix() {
+    let mut testtokn =
+        TestToken::initialized("test_secret_key_attributes_matrix", None);
```

Comment:
the name is confusing as it says "secret key", while it is using also public/private key pairs.

Should we name it rather "sensitive_attributes_matrix"?

Reply:
good point